recipes/glia/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+git clone --recursive https://github.com/ekg/glia.git && \
```

Comment:
Is this really needed? Afaik conda clones the repo for you. Try just `make`. chmod is also probably not needed.